src/webhooks/handleCheckoutSessionCompleted.ts

```diff
@@ -111,7 +111,8 @@ export default async function handleCheckoutSessionCompleted(
   }
 
   if (product.metadata?.type === 'business') {
-    const address = customer.address?.line1 || undefined;
-    await usersService.initializeWorkspace(user.uuid, Number(maxSpaceBytes), address);
+    const amountOfSeats = lineItems.data[0]!.quantity ?? parseInt(price.metadata.minimumSeats);
```

Comment:
Is this the value the customer selects when picking the quantity of users for this plan? This seems like information related to min/max seats per product rather than the customer's picked value

Reply:
It is the amount the user selected/modified during checkout, I added `?? parseInt(price.metadata.minimumSeats);` in case that came out as undefined but now that I think of it that, should never happen. I'll change this to a non null assertion.